src/System.Console/src/System/TermInfo.cs

```diff
@@ -107,6 +107,10 @@ internal sealed class Database
             private readonly int _stringSectionNumOffsets;
             /// <summary>The number of bytes in the strings table of the database.</summary>
             private readonly int _stringTableNumBytes;
+            /// <summary>Whether or not to read the number section as 32-bit integers.</summary>
```

Comment:
@wtgodbe @joperezr looks like we need to add System.Console to the package build list for the 1.0 and 1.1 branches. For 2.0 it is in the platform package so it should be fine there.